lib/ansible/galaxy/api.py

```diff
@@ -914,7 +914,7 @@ def get_collection_signatures(self, namespace, name, version):
         """
         Gets the collection signatures from the Galaxy server about a specific Collection version.
 
-        :param namespace: The collection namespace.
+        :param namespace: The collection namespace
```

Comment:
This is the only change in the diff, and it doesn't do what the PR title says FYI. It's not acceptable in the current form.